vimiv/api/settings.py

```diff
@@ -470,6 +470,9 @@ class thumbnail:  # pylint: disable=invalid-name
     """Namespace for thumbnail related settings."""
 
     size = ThumbnailSizeSetting("thumbnail.size", 128, desc="Size of thumbnails")
+    save = BoolSetting(
+        "thumbnail.save", True, desc="Save new thumbnails to disk for later use"
```

Comment:
Maybe include the shared icon cache somehow?

Save new thumbnails to disk in the shared icon cache?

Reply:
Rephrased this description. Please let me know if it needs any further changes.

Reply:
Thanks! Not sure about "to the disk" compared to "to disk", but can also just change this after merge along with the typo you pointed out.